fsw/src/cs_table_processing.c

```diff
@@ -109,6 +109,7 @@ int32 CS_ValidateEepromChecksumDefinitionTable(void *TblPtr)
                                   "Eeprom Table Validate: Illegal State Field (0x%04X) found in Entry ID %d",
                                   (unsigned short)StateField, (int)OuterLoop);
                 Result = CS_TABLE_ERROR;
+                CS_DisableAllCSCmd()
```

Comment:
```suggestion
                CS_DisableAllCSCmd();
```

Reply:
That wasn't supposed to be in there (not sure how it got pasted in...).
This PR is only for Event ID updates.
Have removed the change to cs_table_processing.c